contrib/experimental/great_expectations_experimental/expectations/expect_column_minimum_bounding_radius_to_be_between.py

```diff
@@ -299,4 +299,4 @@ def _validate(
 
 
 if __name__ == "__main__":
-    ExpectColumnMininumBoundingRadiusToBeBetween().print_diagnostic_checklist()
+    ExpectColumnMinimumBoundingRadiusToBeBetween().print_diagnostic_checklist()
```

Comment:
not seeing what the delta is here. They look identical.

Reply:
numBound vs mumBound